src/index.js

```diff
@@ -32,8 +32,8 @@ app.use(session({
 app.use(passport.initialize());
 app.use(passport.session());
 app.use(routes);
-// cron from deleting blacklist tokens
-cron.schedule('* * * * *', () => {
+// cron fro deleting blacklist tokens
```

Comment:
manage your grammar properly. remove commented out code

Reply:
On this side, I don't find commented out code??